pkg/gui/gui.go

```diff
@@ -337,6 +341,23 @@ func (gui *Gui) layout(g *gocui.Gui) error {
 		}
 	}
 
+	if check, _ := g.View("pushPassUname"); check == nil {
```

Comment:
if we're never going to destroy this view (which I think is the case looking at most of this code), we can add a function in view_helpers.rb called gui.getPushPassUnameView(). Although that's a pretty long name for a function. Maybe we should be calling this view the `credentials` view. I'm not fussed though :)